tests/pagination/src/commonMain/graphql/schema.graphqls

```diff
@@ -1,5 +1,14 @@
-type Query {
-  users(first: Int = 10, after: String = null, last: Int = null, before: String = null): UserConnection!
+type Query
+@typePolicy(embeddedFields: "usersCursorBased, usersOffsetBasedWithPage")
```

Comment:
Do you need to embed `usersOffsetBasedWithPage ` here?

Reply:
It's not obvious, but if we don't, we'll end up having to merge these 2 records:
`usersOffsetBasedWithPage` -> `{"users": ["cacheReference(User:42)", "cacheReference(User:43)"]}`
`usersOffsetBasedWithPage` -> `{"users": ["cacheReference(User:44)", "cacheReference(User:45)"]}`

but without any metadata.

What we need instead is to merge:
`ROOT_KEY` -> `{"usersOffsetBasedWithPage": {"users": ["cacheReference(User:42)", "cacheReference(User:43)"]}, (...)}` (+ metadata for `usersOffsetBasedWithPage`)
`ROOT_KEY` -> `{"usersOffsetBasedWithPage": {"users": ["cacheReference(User:44)", "cacheReference(User:45)"]}}` (+ metadata for `usersOffsetBasedWithPage`)

Hope it makes sense?

Also, maybe there's a problem there because I think it means that users will need to do this for all the query fields that paginate (at least with Relay style pagination). For some APIs that's probably a lot!